config/zones/SK.yaml

```diff
@@ -55,13 +55,19 @@ emissionFactors:
       - datetime: '2022-01-01'
         source: EU-ETS, ENTSO-E 2022
         value: 1300.76
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 764.9
     gas:
       - datetime: '2021-01-01'
         source: EU-ETS, ENTSO-E 2021
         value: 347.5807988
       - datetime: '2022-01-01'
         source: EU-ETS, ENTSO-E 2022
         value: 347.58
+      - datetime: '2023-01-01'
+        source: EU-ETS, ENTSO-E 2023
+        value: 514.8
```

Comment:
The power plant *"SPP Kompresorová stanica, Trakovice"* has increased emissions from 93,258 in 2022 to 338,664 in 2023 (3.6x increase).

Generation for this plant has also increased from 254,104 MWh to 657,850 MWh in 2023.